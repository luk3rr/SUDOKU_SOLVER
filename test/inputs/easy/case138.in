000006970 400000020 000020500 970260800 020000460 150704000 083000710 509670083 007839600 
