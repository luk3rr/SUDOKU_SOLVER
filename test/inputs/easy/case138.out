Solving the following grid:
0 0 0 | 0 0 6 | 9 7 0 
4 0 0 | 0 0 0 | 0 2 0 
0 0 0 | 0 2 0 | 5 0 0 
------+-------+------
9 7 0 | 2 6 0 | 8 0 0 
0 2 0 | 0 0 0 | 4 6 0 
1 5 0 | 7 0 4 | 0 0 0 
------+-------+------
0 8 3 | 0 0 0 | 7 1 0 
5 0 9 | 6 7 0 | 0 8 3 
0 0 7 | 8 3 9 | 6 0 0 


Solution found :')

8 3 2 | 1 5 6 | 9 7 4 
4 6 5 | 3 9 7 | 1 2 8 
7 9 1 | 4 2 8 | 5 3 6 
------+-------+------
9 7 4 | 2 6 3 | 8 5 1 
3 2 8 | 9 1 5 | 4 6 7 
1 5 6 | 7 8 4 | 3 9 2 
------+-------+------
6 8 3 | 5 4 2 | 7 1 9 
5 4 9 | 6 7 1 | 2 8 3 
2 1 7 | 8 3 9 | 6 4 5 

Algorithm: BFS
Total time: 25 ms
Total expanded states: 4255
